source/_integrations/emulated_hue.markdown

```diff
@@ -172,6 +176,10 @@ sudo setcap 'cap_net_bind_service=+ep' /srv/homeassistant/homeassistant_venv/bin
 
 Please note that your path may be different depending on your installation method. For example, if you followed the [Virtualenv instructions](/docs/installation/virtualenv/), your path will be `/srv/homeassistant/bin/python3`.
 
+##### Hassio
+
+Nothing special to do on Hassio, the server is already enabled to serve on port 80
```

Comment:
The documentation should be adjusted to handle the Home Assistant OS, Supervised & Container installations by default (as those are the most used installation types).

Additionally, `Hassio` isn't a term used anymore (for a long time already). It is ambiguous.